File: aoc_day0.py
with open("day0_input.txt", "r") as file:
    str0 = file.read()
c = 0
arr0 = []
for i in str0.splitlines():
    try:
        c += int(i)
    except :
        arr0 += [c]
        c = 0   


print(sorted(arr0))
print(69921+70450+72718)

File: day0/aoc_day0.go
package main

import (
	"fmt"
	"sort"
	"strconv"
	"strings"
	"io/ioutil"
    	"log"
)

func main() {
	content, err0 := ioutil.ReadFile("day0_input.txt")
     	if err0 != nil {
          log.Fatal(err0)
     	}
    	str0 :=  string(content)
	var arr0 []int
	c := 0
	for _,val:= range strings.Split(str0, "\n"){
		i, err := strconv.Atoi(strings.TrimSpace(val))
		if err != nil {
			arr0 = append(arr0, c)
			c = 0
			continue
		} else {
			c += i
		}		
	}
	sort.Ints(arr0)
	fmt.Println(arr0[len(arr0)-1], arr0[len(arr0)-1]+arr0[len(arr0)-2]+arr0[len(arr0)-3])
}
